lib/tsconfig.json

```diff
@@ -25,5 +25,5 @@
     "strictPropertyInitialization": true,
     "target": "ES2017"
   },
-  "include": ["src", "test"]
+  "include": ["facade", "src", "test", "tdf3"]
```

Comment:
it would be nice to remove test from bundle

Reply:
This is for running tsc. Bundling is accomplished with web pack.config, which I think doesn't include these? (It gives an entry point, and then tests/etc should be shaken out), or the package.json `files` directive, which removes them from the esm module that is the only thing that seems to be built.